### PR TITLE
[Refactor] Move `default_scheduler_interval_millisecond` from `FeConstants` to `Config`

### PR DESCRIPTION
**Pull request by jiacheng-celonis**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

We would like to customize the `default_scheduler_interval_millisecond` to enable faster schema changes (ALTER TABLEs).

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by decster** [comment]
Maybe also add a new config just for Alter, so changing it won't change other components' behavior?

File: fe/fe-core/src/main/java/com/starrocks/common/Config.java [hunk 1]
```diff
@@ -2236,4 +2236,11 @@ public class Config extends ConfigBase {
 
     @ConfField
     public static boolean enable_execute_script_on_frontend = true;
+
+
+    /**
+     * The default scheduler interval for alter jobs and routine jobs.
+     */
+    @ConfField(mutable = true)
+    public static int default_scheduler_interval_millisecond = 10000;
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/common/Config.java:2245]:
```suggestion
    public static int scheduler_interval_seconds = 10;
```
is a better name. The scheduler interval is not less than one second.

Comment (decster) [reply fe/fe-core/src/main/java/com/starrocks/common/Config.java:2245]: To speedup UT, it may be set to a very small value?

Comment (jiacheng-celonis) [reply fe/fe-core/src/main/java/com/starrocks/common/Config.java:2245]: Yeah, some unit tests overwrite the previous value to 1 millisecond.

File: fe/fe-core/src/main/java/com/starrocks/common/Config.java [hunk 2]
```diff
@@ -2236,4 +2236,17 @@ public class Config extends ConfigBase {
 
     @ConfField
     public static boolean enable_execute_script_on_frontend = true;
+
+
+    /**
+     * The default scheduler interval for alter jobs.
+     */
+    @ConfField(mutable = true)
+    public static int default_alter_scheduler_interval_millisecond = 10000;
```
Comment (decster) [inline fe/fe-core/src/main/java/com/starrocks/common/Config.java:2245]: After making it configurable, we can remove `default_` prefix?

Comment (jiacheng-celonis) [reply fe/fe-core/src/main/java/com/starrocks/common/Config.java]: Done

**Review by decster — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 4 / 4 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/Config.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadScheduler.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/alter/AlterHandler.java|1|1|100.00%|[]|

**Review by chaoyli — APPROVED** [review]

**Pull request merged**